examples/llama2_inference/stateless_llama.py

```diff
@@ -399,39 +388,54 @@ def run_vmfb_comparison(args):
     example_input_id = initial_input.input_ids
     device_inputs = [ireert.asdevicearray(config.device, example_input_id)]
 
-    step0 = ModuleCompiled["get_seq_step"]() 
-    print("step0 :"+str(step0))
+    step0 = ModuleCompiled["get_seq_step"]()
     results = ModuleCompiled["run_initialize"](*device_inputs)
 
-    pkv = ModuleCompiled["get_global_state"]().to_host() 
-    step = ModuleCompiled["get_seq_step"]() 
-    print(f"step: {step}")
-    sliced = pkv[0,:,:,:step,:]
+    pkv = ModuleCompiled["get_global_state"]().to_host()
+    step = ModuleCompiled["get_seq_step"]()
+    summed = torch.sum(torch.tensor(pkv))
+    sliced = pkv[0, :, :, :step, :]
 
     def format_out(results):
         return torch.tensor(results.to_host()[0][0])
 
-#    print(tokenizer.decode(format_out(results)))
-    for i in range(10):
-        results = ModuleCompiled["run_forward"](results)
-        step = ModuleCompiled["get_seq_step"]() 
-        print(f"step: {step}")
-        print(tokenizer.decode(format_out(results)))
     model = InferenceModel(args)
 
-
     def get_token_from_logits(logits):
         return torch.argmax(logits[:, -1, :], dim=1)
 
     base_model_results = model.base_model.forward(example_input_id)
     base_model_token = get_token_from_logits(base_model_results.logits)
-#    print(tokenizer.decode(base_model_token))
+    bm_pkv = base_model_results.past_key_values
+    all_ours = []
```

Comment:
Consider renaming to turbine_results and torch_results, or example_results and reference_results.